src/locales/default.json

```diff
@@ -87,6 +87,7 @@
   "removeSpace": "for the space {0}",
   "noVotingPower": "Oops, it seems you don't have any voting power at block {blockNumber}.",
   "quorumReached": "quorum reached",
+  "quorumRejection": "quorum rejection",
```

Comment:
Please don't use translation files anymore. We will have harder time when migrating to SX